Locales.lua

```diff
@@ -631,7 +631,7 @@ L["You entered a achievement ID that is already being used by another item."] =
 L["Instance Difficulty"] = true
 L["Determines which instance difficulties this item may be obtained in. Leave everything unchecked if the instance difficulty doesn't matter.\n\nIf you specified a Statistic ID for this item, the Instance Difficulty is probably meaningless, because all modern statistics already incorporate the difficulty.\n\nYou can check multiple items in this list at once."] = true
 L["Take screenshots"] = true
-L["When on, Rarity will take a screenshot when the achievement alert pops up indicating that you obtained an item."] = true
```

Comment:
Won't this break the ``Options.lua`` entry ``takeScreenshot`` unless the text provided there is also changed?

Reply:
Yep, you are absolutely right. Having the strings in multiple locations and not as a single key-value pair still trips me up it seems. Updated the missing one now.